jovian/utils/clone.py

```diff
@@ -71,7 +80,14 @@ def clone(slug, version=None, fresh=True, include_outputs=True, overwrite=False)
     # Download gist metadata
     ver_str = '(version ' + str(version) + ')' if version else ''
     log('Fetching ' + slug + " " + ver_str + "..")
+    
+    # Try to get gist with current user 
     gist = get_gist(slug, version, fresh)
+
+    # If no gist retrieve
```

Comment:
Can remove this comment. Can restrict comments to 
- complex part to summarize the functionality 
- for something which may seem unusual at first glance 
- to group certain things under a header comment (might see in webapp constant files)

Reply:
Removed.